packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -71,11 +72,10 @@ export const BaseButtonStyles = css`
         background-color: ${neutralFillActiveBehavior.var};
     }
 
-    .control:${focusVisible} {
```

Comment:
Doh - I just realized this is going to break `:focus-visible` when that rolls in. :focus-visible is not out from behind a flag yet so I *think* this is okay for now but we should get other folk's opinion on that. @EisenbergEffect @chrisdholt

Reply:
Hmm. Assuming we had focus visible, what would be the alternative fix? If we can plan for forward compat, that's definitely preferred.

Reply:
Propose `:focus-visible-within` be added to the spec? 🙃 I can't think of a pure CSS way to keep `:focus-visible`. Does fast have a mechanism for forwarding styles to child elements?

Reply:
[It's been discussed!]( https://github.com/WICG/focus-visible/issues/151)

I can't either... You can forward styles to shadowed element has a [part attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/part), which button does.

Reply:
@nicholasrice @CuddleBunny `:focus-visible` is shipping in Chrome 86 which is currently in Beta: https://twitter.com/ChromiumDev/status/1301262894617640960

This will cover both Chrome and I'm checking on if it will also be in "Edgium" (h/t @CuddleBunny) for 86...

Reply:
Perhaps this should just be closed as a non-issue and documented that button padding can be controlled by targeting the control part? At this point it is down to developer experience vs accessibility. The latter is probably more important.

Reply:
@CuddleBunny Unfortunately I think that is the best course for now. Thanks for looking at this though!